go.mod

```diff
@@ -15,6 +15,5 @@ require (
 	github.com/spf13/cobra v0.0.6
 	github.com/stretchr/testify v1.4.0
 	github.com/trustbloc/edge-core v0.1.2
-	github.com/trustbloc/edv v0.1.2
-	github.com/trustbloc/sidetree-core-go v0.1.2-0.20200226223710-852e23076586
```

Comment:
I ran a `go mod tidy` and it removed that sidetree-core-go line. I guess it was left over from a previous commit.